flowpipe/plug.py

```diff
@@ -293,6 +291,7 @@ def __init__(self, key, node, parent_plug, value=None):
         super(SubInputPlug, self).__init__(
             '{0}.{1}'.format(parent_plug.name, key), node,
             (OutputPlug, SubOutputPlug))
+        self.key = key
```

Comment:
Not necessary right now, but if we add more sub-plug functionality, we should consider adding a new base class `SubPlug` and have `SubInputPlug`/`SubOutputPlug` inherit from both `IPlug` and `SubPlug`, or have multiple inheritance layers.